runner/query_run.go

```diff
@@ -39,85 +39,74 @@ func SetupQueryRunnerForAllServers(ctx context.Context, servers []*state.Server,
 	}
 }
 
+func runQueryOnDatabase(ctx context.Context, server *state.Server, collectionOpts state.CollectionOpts, logger *util.Logger, id int64, query *state.QueryRun) (string, error) {
```

Comment:
Nitpick, but I find it confusing to use C-style function ordering where the entrypoints to a file are at the bottom of a file. Go doesn't impose function ordering requirements like C, so I would expect this function to be below the top-level `run` function

Reply:
Fair point, I can reorder.